src/js/components/FormField/FormField.js

```diff
@@ -241,6 +263,20 @@ const FormField = forwardRef(
       return readOnly;
     }, [children]);
 
+    const wantFocusIndicator = useMemo(() => {
```

Comment:
```suggestion
    const containerFocus = useMemo(() => {
```

Reply:
Trying to align logic with theme namespace.

Reply:
yeah good point

Reply:
Although wondering if maybe a better name could be chosen? Just a suggestion to try to minimize all the different terms.